src/service.ts

```diff
@@ -189,19 +204,13 @@ export class Service {
         ]
       };
       const input = encodeURIComponent(JSON.stringify(project)).replace("%20", "+");
-      return this.sendRequest("input=" + input + "&action=build");
+      return this.sendRequest("input=" + input + "&action=build", "service");
```

Comment:
Instead of passing "rustc" or "service" as a string, please introduce an enumeration.